docs/03-concepts/01-actor-lifecycle.mdx

```diff
@@ -0,0 +1,115 @@
+---
+title: Actor lifecycle
+sidebar_label: Actor lifecycle
+---
+
+## Lifecycle methods
+
+At the start of its runtime, the actor needs to initialize itself, its event manager and its storages,
+and at the end of the runtime it needs to close these cleanly.
+The Apify SDK provides several options on how to manage this.
+
+### `Actor.init()` and `Actor.exit()`
+
+The [`Actor.init()`](../../reference/class/Actor#init) method initializes the actor,
+the event manager which processes the actor events from the platform event websocket,
+and the storage client used in the execution environment.
+It should be called before performing any other actor operations.
+
+The [`Actor.exit()`](../../reference/class/Actor#exit) method then exits the actor cleanly,
+tearing down the event manager and the storage client.
+There is also the [`Actor.fail()`](../../reference/class/Actor#fail) method, which exits the actor while marking it as failed.
+
+```python title="src/main.py"
+from apify import Actor
+from apify.consts import ActorExitCodes
+
+async def main():
+    await Actor.init()
+    try:
+        print('Actor input:', await Actor.get_input())
```

Comment:
Just idea to use our logger instead of print.

Reply:
Yeah, good point, I'll rewrite the `print`s to `Actor.log.xxx` where appropriate